recipe/meta.yaml

```diff
@@ -61,6 +61,9 @@ outputs:
         - {{ pin_subpackage("libcxx", exact=True) }}
       run:
         - {{ pin_subpackage("libcxx", exact=True) }}
+      run_constrained:
+        - clang {{ bootstrap_version }}
+        - llvm {{ bootstrap_version }}
```

Comment:
This should probably be `version` rather than `bootstrap_version` at this time, if the toolchain is fully rebuilt. Same with the other places.